src/rer/newsletter/browser/channel/users/usersimport.py

```diff
@@ -15,6 +15,7 @@
 import csv
 import re
 from six.moves import range
+from six import PY3
```

Comment:
Don't import PY3 and check for it..it's better to use PY2 and do your conditions for PY2 only, assuming that PY3 is the default.

In this way, when we drop python2 compatibility, it will be easier to cleanup the code